examples/ex_cpu.c

```diff
@@ -0,0 +1,92 @@
+/* An example showing the use of al_get_cpu_count() and al_get_ram_size(). */
```

Comment:
I think it'd be sufficient to just use the log for this addon, and not bother with a display.